### ISSUE TITLE
v1.3.4 checksum list has unexpected SHA-256 hash

### ISSUE THREAD

**Issue by rhernaus** [lord-kyron/terraform-provider-phpipam#48]
Since a couple of hours there is a checksum mismatch for version 1.3.4.

```
$ terraform init
Initializing modules...
...

Initializing provider plugins...
...
- Finding lord-kyron/phpipam versions matching ">= 1.2.8"...
- Installing lord-kyron/phpipam v1.3.4...

Partner and community providers are signed by their developers.
If you'd like to know more about provider signing, you can read about it here:
https://www.terraform.io/docs/cli/plugins/signing.html

╷
│ Error: Failed to install provider
│
│ Error while installing lord-kyron/phpipam v1.3.4: checksum list has unexpected SHA-256 hash 02109ac8036feddb445e6459bf1e509a59f4cc6650960a2bd27d222891ee2e49 (expected ded569ed8c0d0722dc2aba69fc35c8839c952455b7dfd31f2f83e0402a87d357)
╵
```

**Comment by pavel-z1** [comment]
Hi @lord-kyron 
Seems, we have issue with latest release version 1.3.4.

Error with hash during terraform init:
```
│ Error: Failed to install provider
│
│ Error while installing lord-kyron/phpipam v1.3.4: checksum list has unexpected SHA-256 hash 20cc5b875db78014a05211a6018dddfcce4058f5da75aeb5994c84c9bd7f48b7 (expected f3175d7d9ba103d24925e37ebd7079b9fac6022369e72e66f66a77d186d56ed8)
```

Can you please redeploy binary to the registry.terraform.io?

**Comment by lord-kyron** [comment]
Yes, sorry for that!

**Comment by lord-kyron** [comment]
@pavel-z1 @rhernaus 
Issue should be fixed by the latest commit - v1.3.6

**Comment by rhernaus** [comment]
Working again for me. Thanks @lord-kyron
